vignettes/views.Rmd

```diff
@@ -89,11 +89,22 @@ data["class"] <- c("V", "VI")
 synStore(Table(view$properties$id, data))
 ```
 
-The change in annotations reflect in synGetAnnotations():
+The change in annotations is reflected in synGetAnnotations():
 ```{r collapse=TRUE, eval=FALSE}
 synGetAnnotations(file2$properties$id)
 ```
 
+A unique etag is associated with every file that updates when changes are made to the file annotations or metadata. Any updates pushed to Synapse will change an object's etag.
```

Comment:
the etag changes when any changes are made to a file, including the contents, annotations, or metadata (I suppose metadata includes contents as the file handle would change, but might be better to frame it this way!)